modules/sequence/src/main/scala/lucuma/odb/sequence/gmos/longslit/Science.scala

```diff
@@ -658,7 +658,7 @@ object Science:
           val blocksʹ     = blocks.updatedUnsafe(pos, block.add(n))
           ((blocksʹ, aix + 1, csʹ), atom)
         }
-        .cons1((blocks, 0, 0), atom0) // put the first atom back
+        .cons1((blocks, 0, TimeEstimateCalculator.Last.empty), atom0) // put the first atom back
```

Comment:
Where did this change come from?

Reply:
The type of that parameter so far is `TimeEstimateCalculator.Last` and Scala 3.6.2 (incorrectly) complains that the subsequent pattern match does not contemplate all cases.

The value of that parameter is ignored, I just set it to something that matched the type thus far to silence the warning.